### PR TITLE
Static parameter warning

### PR DESCRIPTION
**Pull request by ivanpesin**
I have added the logic to produce log warning if static command line parameters are specified in 'execute-command' (issue #151):

`"execute-command": "/bin/echo test"`

will produce:

```
[webhook] 2017/08/25 23:31:55 unable to locate command: '/bin/echo test'
[webhook] 2017/08/25 23:31:55 use 'pass-arguments-to-command' to specify args for '/bin/echo'
```

Let me know if you want me to do it in a different way.

### CODE REVIEW

File: webhook.go [hunk 1]
```diff
@@ -312,7 +312,21 @@ func hookHandler(w http.ResponseWriter, r *http.Request) {
...
 	var errors []error
 
-	cmd := exec.Command(h.ExecuteCommand)
+	// check the command exists
+	cmdPath, err := exec.LookPath(h.ExecuteCommand)
+	if err != nil {
+		log.Printf("unable to locate command: '%s'", h.ExecuteCommand)
+
+		// check if parameters specified in execute-command by mistake
+		if strings.IndexByte(h.ExecuteCommand, ' ') != -1 {
```
Comment (gdubicki) [inline webhook.go]: This way of checking it will make it impossible to use commands with spaced in their names, even if they are quoted properly with `\`.

Comment (gdubicki) [reply webhook.go]: Sorry, nvm, it will not.

**Comment by adnanh** [comment]
Looks great! If you could add some tests for this feature, I would be more than glad to merge it in! Thanks.

**Comment by ivanpesin** [comment]
@adnanh I've added a test for this feature -- let me know if they look fine.

Thanks.

**Comment by simplenotezy** [comment]
It's not documented anywhere how to use "pass-arguments-to-command"

**Comment by moorereason** [comment]
@simplenotezy,
In the future, please don't comment on issues that were closed years ago.  Open a new issue.

To answer your question, see the Github example at https://github.com/adnanh/webhook/blob/master/docs/Hook-Examples.md#incoming-github-webhook

**Pull request merged**